src/test/ui/align-with-extern-c-fn.rs

```diff
@@ -10,6 +10,7 @@
 #[repr(align(16))]
 pub struct A(i64);
 
+#[allow(improper_ctypes)]
```

Comment:
It might make sense to put `repr(C)` on `A` but I'm not sure. Does anyone else know?

Reply:
I've added a commit which changes this test.